dnf/cli/commands/shell.py

```diff
@@ -119,9 +119,8 @@ def onecmd(self, line):
                     cmd.cli.demands = copy.deepcopy(self.cli.demands)
                     cmd.configure()
                     cmd.run()
-                except dnf.exceptions.Error as e:
-                    logger.error(_("Error:") + " " + e.value)
-                except:
+                except Exception as e:
```

Comment:
What about to replace `except Exception as e:` by `except dnf.exceptions.Error as e:`? Or is there any reason why tolerate standard Python exceptions?

Reply:
It's the same situation as when invoking dnf command from bash. You do not want to see traceback but an error message instead. And there could occur also other exception during command execution - IO errors, Runtime errors...
And it would be regression - previous behaviour was to catch any errors (although only dnf.Errors were presented to user, which I consider bad).